spec/controllers/service_provider_controller_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+describe ServiceProviderController do
+  include SamlAuthHelper
+
+  describe '#update' do
+    let(:dashboard_sp_issuer) { 'some-dashboard-service-provider' }
+    let(:array_of_sps) do
```

Comment:
I would probably just call this `service_providers` or something to avoid the acronym and avoid using the object type in the name https://github.com/thoughtbot/guides/tree/master/style#naming